src/drivers/Akamai.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace ostark\upper\drivers;
+
+use GuzzleHttp\Exception\BadResponseException;
+use ostark\upper\exceptions\AkamaiApiException;
+
+/**
+ * Class Akamai Driver
+ *
+ * @package ostark\upper\drivers
+ *
+ */
+class Akamai extends AbstractPurger implements CachePurgeInterface
+{
+    /**
+     * @var string
+     */
+    public $host;
+
+    /**
+     * @var string
+     */
+    public $clientToken;
+
+    /**
+     * @var string
+     */
+    public $clientSecret;
+
+    /**
+     * @var string
+     */
+    public $accessToken;
+
+    /**
+     * @var string
+     */
+    public $maxSize;
+
+    /**
+     * Purge cache by tag
+     *
+     * @param string $tag
+     *
+     * @return bool
+     */
+    public function purgeTag(string $tag)
+    {
+        if ($this->useLocalTags) {
+            return $this->purgeUrlsByTag($tag);
+        }
+
+        $this->sendRequest('production', 'POST', 'tag', $tag);
+        $this->sendRequest('staging', 'POST', 'tag', $tag);
```

Comment:
IMO, too much hard coding here

Reply:
What would you like to see differently? Akamai knows only production/staging, it's not something you can name yourself.

Reply:
OK, I didn't know